src/parser/mod.rs

```diff
@@ -6083,6 +6083,21 @@ impl<'a> Parser<'a> {
                 None
             };
             Ok(Some(ColumnOption::Identity(property)))
+        } else if dialect_of!(self is SQLiteDialect)
+            && self.parse_keywords(&[Keyword::ON, Keyword::CONFLICT])
+        {
+            // Support ON CONFLICT for SQLite
+            let on_conflict = self.parse_one_of_keywords(&[
+                Keyword::ROLLBACK,
+                Keyword::ABORT,
+                Keyword::FAIL,
+                Keyword::IGNORE,
+                Keyword::REPLACE,
+            ]);
+            match on_conflict {
+                Some(keyword) => Ok(Some(ColumnOption::OnConflict(keyword))),
+                _ => unreachable!(),
```

Comment:
Ah so this behavior doesn't seem desirable, if we get wrong sql the parser shouldnt panic? I think rather the code would be roughly e.g.
```rust
Ok(Some(ColumnOption::OnConflict(self.expect_one_of_keywords(&[...])?)))
```

Reply:
I fully agree with the suggestion, I adapted the code accordingly